ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/wizards/templates/PreprocessorParser.java

```diff
@@ -212,14 +212,12 @@ private void parse() throws Exception {
 				continue;
 			}
 
-			int opcode = 0;
-
-			opcode = switch (token) {
+			int opcode = switch (token) {
 				case T_AND -> opcode = OP_AND;
```

Comment:
I think this assignment is not intentional.

Reply:
@merks - agree, looks like accidentally retained in the previous change. fixed it now.